icu-date.lua

```diff
@@ -191,6 +191,21 @@ function _M:format(format)
   return uchar_to_string(result, result_length)
```

Comment:
It's time to add testing against different libicu versions?

I see the [icu4c-71_1-Ubuntu20.04-x64.tgz](https://github.com/unicode-org/icu/releases/download/release-71-1/icu4c-71_1-Ubuntu20.04-x64.tgz) archive on the [release page](https://github.com/unicode-org/icu/releases/tag/release-71-1), it should be suitable here.

Reply:
I think yes. Since "version detection" logic appeared.

Reply:
Pushed https://github.com/tarantool/icu-date/pull/31/commits/9485d3a204af4e07d0faf384c87e24c23ee5f5bc

However my approach requires tarantool to be built from source. Also libicu is built from source too. Probably I started to use `ldconfig` too late during testing.

Reply:
I don't mind building from source, it is definitely better than lack of such testing.

I would keep testing with distribution default icu and keep this testing fast (using setup-tarantool) to receive fast feedback during development. How about moving testing against different icu versions into its own workflow?

Reply:
I would also appreciate ideas how to speed up it if you have some. You can add TODO comments right into the workflow file.

Reply:
> I don't mind building from source, it is definitely better than lack of such testing.

I found a way how to avoid building libicu. Your idea with [icu4c-71_1-Ubuntu20.04-x64.tgz](https://github.com/unicode-org/icu/releases/download/release-71-1/icu4c-71_1-Ubuntu20.04-x64.tgz) works. I just missed to use ldconfig.

> I would also appreciate ideas how to speed up it if you have some. You can add TODO comments right into the workflow file.

After previous message I even don't know. Maybe we could download Tarantool EE with different bundled libicu. But probably it's not completely fair way.

Reply:
> this patch manually build tarantool and libicu.

Forgotten in the commit message.

> After previous message I even don't know. Maybe we could download Tarantool EE with different bundled libicu. But probably it's not completely fair way.

Sure, it is irrelevant now.